package.json

```diff
@@ -7,10 +7,9 @@
     "karma": "karma start",
     "test:server": "mocha --recursive ./test/server ./test/common",
     "lint": "eslint --ext .js --ext .jsx lib src test",
-    "prestart": "npm run webpack-prod",
     "start": "node src/server.js",
     "test": "npm run lint && npm run test:server && npm run karma -- --single-run",
-    "watch": "npm run karma & BABEL_ENV=hot npm start & npm run watch-lint & npm run test:server",
+    "watch": "npm run karma & npm run webpack && BABEL_ENV=hot npm start & npm run watch-lint & npm run test:server",
```

Comment:
You shouldn't need to change this line.

BABEL_ENV=hot turns on the webpack-dev-server path in server.js which runs webpack using the dev server writing to an in-memory filesystem.

What the above will do is it will run webpack once to regenerate the files in /generated on the invocation of `npm run watch` but the server will run webpack a second time for the in memory filesystem.  Then as we change source code, the in-memory filesystem will update inside the webpack-dev-server, but the on-disk version will not.